xml/ay_kernel_dumps.xml

```diff
@@ -90,6 +90,18 @@
 &lt;/kdump&gt; </screen>
    </example>
 
+   <para>
+    Kdump is enabled by default. The following configuration show how it disable it.
```

Comment:
Well, that's not 100% accurate. For instance, in openSUSE (Tumbleweed and Leap) is only enabled for S/390 architectures. And regarding SLE, it is not enabled on aarch64 (not available) and on those systems with less than 1GB of RAM. Yes, I know, it sounds confusing.

I am not sure whether it is a good idea to include all those details, but as this is the source for openSUSE documentation too, I guess we should rephrase it a little bit. If you prefer to have separate texts, you can use the "os" attribute.

```xml
<para os="sles,sled">
 bla bla
</para>
<para os="osuse">
 bla bla
</para>
```

Reply:
About the second part, you could write: "The following example shows how to disable it".

Reply:
ok, if the technical bits are OK then I will fiddle with the wording.

Reply:
I need some clarification on the supported kdump platforms and architectures. There is already a Note at the beginning of the kdump section that says " This feature is not available on the IBM Z (s390x) architecture."

Does this wording work?

```   <note>
    <title>Unsupported kdump platforms and architectures</title>
    <para>
     kdump is not available on the &zseries; (s390x) architecture.
    </para>
    <para>
      kdump is not available on &aarch64;.
    </para>
    <para>
      It is not available on systems with less than 1GB of memory.
    <para>      
   </note>
...

   <para>
    Kdump is enabled by default. The following configuration shows how to disable it.
   </para>

   <example>
    <title>Disable Kdump configuration</title>
<screen>&lt;kdump&gt;
  &lt;add_crash_kernel config:type="boolean"&gt;false&lt;/add_crash_kernel&gt;
&lt;/kdump&gt; </screen>
   </example>

Reply:
@imobachgs can you clarify platform support? Thanks!